src/server/src/main/java/io/cassandrareaper/core/GenericMetric.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2017-2017 Spotify AB
+ * Copyright 2017-2018 The Last Pickle Ltd
```

Comment:
if this is a new file the copyright lines should be
```
 *
 * Copyright 2019-2019 The Last Pickle Ltd
```

And other touched files need to the second line to be updated to end with `-2019 The Last Pickle Ltd`